integration/confidential/lib.sh

```diff
@@ -223,28 +223,10 @@ setup_common_signature_files_in_guest() {
 
 setup_offline_fs_kbc_signature_files_in_guest() {
 	setup_common_signature_files_in_guest
-	setup_offline_fs_kbc_agent_config_in_guest
+	add_kernel_params "agent.aa_kbc_params=offline_fs_kbc::null"
 	cp_to_guest_img "etc" "${SHARED_FIXTURES_DIR}/offline-fs-kbc/aa-offline_fs_kbc-resources.json"
 }
 
-setup_offline_fs_kbc_agent_config_in_guest() {
-	local rootfs_agent_config="/etc/agent-config.toml"
-
-	clone_katacontainers_repo
-	sudo -E AA_KBC_PARAMS="offline_fs_kbc::null" HTTPS_PROXY="${HTTPS_PROXY:-${https_proxy:-}}" envsubst < ${katacontainers_repo_dir}/docs/how-to/data/confidential-agent-config.toml.in | sudo tee ${rootfs_agent_config}
```

Comment:
this function also set the `https_proxy` for guest, could you add this: https://github.com/kata-containers/tests/blob/CCv0/integration/kubernetes/confidential/agent_image.bats#L90 in `agent_image_encrypted.bats`

Reply:
Sorry I missed this Arron - rushing too much here. I've added a new library function to setup the proxy. Longer-term I'd like to get this merged with the crictl approach better to avoid duplicated code, but that is for another day - probably once we've removed skopeo entirely and can simplify things more.